polarityio/phantom
# Phantom Upgrade - Adding the ability to see Playbook Run History and create Events/Containers when searching On Demand

## Steps to Test
1.) Install Integration and add credentials from 1pass
2.) Change the config rejectUnauthorized to false, turn off caching, and restart the Integration
3.) Search these keys and verify the responses in the overlay seem reasonable and contain Playbook Run History that looks correct (reference postman collection as needed):
```
100.100.100.100
192.185.91.42
178.128.76.6
```
4.)  Search `docs.microsoft.com` onDemand and verify the panel that appears in the overlay works and looks correct.
5.) Review code and give feedback on 
- [ ] Formatting/Readability
- [ ] Actual functionality being executed (e.g. Auth, requests, data transformations)


> **Note:** This Integration allows for creating Events/Containers in Phantom when searching for entities On Demand.  It is important that if you create an Event/Container in Phantom while testing, you to delete the newly created Event/Container after you are done.
